bitnami/rabbitmq/values.yaml

```diff
@@ -107,6 +107,10 @@ rabbitmq:
   ## Plugins to enable
   plugins: "rabbitmq_management rabbitmq_peer_discovery_k8s"
 
+
+  ## Community plugins to install and enable
+  communityPlugins: {}
```

Comment:
Should not this be an array? the same applies to the values-production.yaml

Reply:
We need two values for each plugin; 

1. The url to download the zip file
2. The name of the plugin file when we unzip it.

For example the urls might be 

https://bintray.com/rabbitmq/community-plugins/download_file?file_path=3.7.x%2Frabbitmq_delayed_message_exchange%2Frabbitmq_delayed_message_exchange-20171201-3.7.x.zip

However if I don't use a specific name I can't tell what the content of the rabbitmq_delayed_message_exchange-20171201-3.7.x.zip are. There is nothing that guarantees the name would be `abbitmq_delayed_message_exchange-20171201-3.7.`. So basically we need a safe way of renaming the plugin to a predictable name so we can later on call  `rabbitmq-plugins enable <plugin_name>`